uber/config.py

```diff
@@ -110,15 +110,15 @@ def get_attendee_price(self, dt=None):
                 if (dt or sa.localized_now()) >= day:
                     price = bumped_price
 
-                # Only check bucket-based pricing if we're not checking an existing badge AND
-                # we don't have hardcore_optimizations_enabled config on AND we're not on-site
-                # (because on-site pricing doesn't involve checking badges sold).
-                if not dt and not c.HARDCORE_OPTIMIZATIONS_ENABLED and sa.localized_now() < c.EPOCH:
-                    badges_sold = self.BADGES_SOLD
-
-                    for badge_cap, bumped_price in sorted(self.PRICE_LIMITS.items()):
-                        if badges_sold >= badge_cap and bumped_price > price:
-                            price = bumped_price
+            # Only check bucket-based pricing if we're not checking an existing badge AND
+            # we don't have hardcore_optimizations_enabled config on AND we're not on-site
+            # (because on-site pricing doesn't involve checking badges sold).
+            if not dt and not c.HARDCORE_OPTIMIZATIONS_ENABLED and sa.localized_now() < c.EPOCH:
```

Comment:
Our price limit (bucket-based pricing) unit tests were failing, and after some time I figured out that it was because we were only looking at price limits if we had any price bumps (date-based pricing) configured. Whoops!